### ISSUE TITLE
Since update to 4.0.0 and 4.0.1 Torrents added by Sonarr always start off as Paused

### ISSUE THREAD

**Issue by Scabbrox** [transmission/transmission#5005]
### What is the issue?

Under 3.0 transfer added automatically by Sonarr auto started and indeed the automatically start torrent flag is ticked in my settings.

Under 4.0 and now 4.0.1 they are added as paused so don't being download until I notice and un-pause them.  Is this a new feature, bug or is there a setting I haven't been able to find where this behaviour can be changed?

### Which application of Transmission?

macOS app

### Which version of Transmission?

4.0.1

**Comment by ckerr** [comment]
I don't use Sonarrr but I've been meaning to give it a try, so I'll see if I can repro this. @Scabbrox what version of Sonarr are you using?

**Comment by cybersteel8** [comment]
Wasn't this fixed between 4,0.0 and 4.0.1? https://github.com/transmission/transmission/issues/4740

**Comment by Scabbrox** [comment]
> I don't use Sonarrr but I've been meaning to give it a try, so I'll see if I can repro this. @Scabbrox what version of Sonarr are you using?

3.0.9.1549

**Comment by Scabbrox** [comment]
> Wasn't this fixed between 4,0.0 and 4.0.1? #4740

No, I updated today and the first thing it did was add 10 torrents as paused.

**Comment by maxz** [comment]
I have no insights into Sonar but can at least say that magnet links added via a RPC call start properly again for me with version 4.1 whilst they did not with version 4.0.

**Comment by ckerr** [comment]
I haven't had a chance to test yet but from the looks of [this comment](https://github.com/Sonarr/Sonarr/issues/5425#issuecomment-1441880476) Sonarr is using the RPC API.

@maxz which client are you using -- `transmission-daemon`? I'm wondering if it's something specific to macOS + RPC since @Scabbrox indicated they're using the macOS client.

**Comment by maxz** [comment]
@ckerr `transmission-daemon` on Linux.

**Comment by Scabbrox** [comment]
I've not been able to test Sonarr today as hoped as the show that normally would have an episode added this morning is on a skip week and also the RSS trackers have been giving down errors all morning.

Adding a new show works normally as does adding Magnet links so it may be that this is fixed and the issue with the unmet episodes yesterday was some sort of update weirdness that won't be repeated.

I'll come back next week when the next show is due to auto-add from Sonarr and let you know if it's still paused or actually fixed.  Sorry I can't do so before as adding anew show doesn't reproduce the error so could be an indication it is working.

**Comment by meshuggah44** [comment]
I was just about to create the same issue. This has been happening to me since updating to 4.0, be it with Sonarr or Radarr

**Comment by Scabbrox** [comment]
As promised coming back on this.  Today's download was added automatically to the queue by Sonarr and started automatically so it's fixed in 4.0.1 as far as I can tell.

Sorry for the false issue.

**Comment by ckerr** [comment]
@Scabbrox I really appreciate you checking back in -- and with good news, no less!

Thanks for the update :+1:
